### ISSUE TITLE
Documentation is not clear

### ISSUE THREAD

**Issue by silva96** [excid3/noticed#72]
https://github.com/excid3/noticed/blob/d147d49591b38a141b24634fdcbafa4b5e7641f9/lib/noticed/delivery_methods/email.rb#L22

This line is not merging the recipient

https://github.com/excid3/noticed/blob/master/README.md#L220

> Use a custom method to define the params sent to the mailer. recipient will be merged into the params.

I had to merge it myself.

**Comment by excid3** [comment]
Good catch. Want to send over a PR that merges the receipient in both cases?

**Comment by silva96** [comment]
yeah sure, I also found this, don't know if I misunderstood something but, I had to "hack it" too

I was having this error when adding the "unless"
`NoMethodError (undefined method `read?' for #<CommentNotification:0x00007feb4dd71aa0>`

```ruby
# frozen_string_literal: true

class CommentNotification < Noticed::Base
  deliver_by :database
  deliver_by :email, mailer: 'NotificationMailer', format: :format_for_email, delay: 10.minutes, unless: :read?

  param :commented_by
  param :post
  param :comment

  def message
    I18n.t('notifications.comment.body', descriptive_attributes)
  end

  def descriptive_attributes
    {
      commented_by: params[:commented_by],
      post_title: params[:post].title,
      comment_content: params[:comment].content
    }
  end

  def format_for_email
    descriptive_attributes.merge(recipient: recipient)
  end

  delegate :read?, to: :record   # =====> had to add this to avoid getting error.
end
```

**Comment by excid3** [comment]
And you're right about the `read?`, that was actually just an example and wasn't intended to work. Obviously, that would be super handy if it did work, so I'm going to add that. 👍

**Comment by silva96** [comment]
I just was pointing that reading the documentation example looked like it would work out of the box, so at least a clarification would be needed, but if you want to implement it, Nice!!

I think the "read" method is the most common case you would call so it makes totally sense to implement the delegation in the gem.

**Comment by excid3** [comment]
Yeah, it reads so cleanly that way, I have to do it. 😜

Just released v1.2.21 that includes your fix and `read?` `unread?` delegates. 👍 

Appreciate the help!
